### DISCUSSION TITLE
Netlify:  Background functions support?

### DISCUSSION THREAD

**Discussion by MentalGear** [sveltejs/kit#9784] [Q&A]
I have noticed nothing in the guide regarding usage of netlify background function for longer runtimes. Is this possible with sveltekit ?

**Comment by jnschrag** [comment]
This is something that I've recently been looking into as well. I've been running into various issues with trying to use Netlify's built in functions by putting my stuff in `/netlify/functions/hello-world-background.js` and trying to call it from Sveltekit, so being able to use a `+server.js` page for that would be wonderful.
